src/main/java/com/yelp/nrtsearch/server/luceneserver/SimpleCopyJob.java

```diff
@@ -101,7 +101,8 @@ public void start() throws IOException {
 
   @Override
   public void runBlocking() throws Exception {
-    while (visit() == false) ;
+    while (visit() == false)
+      ;
```

Comment:
This formatting seems weird. I would prefer having the semi-colon in line with while only. If not maybe we can use empty braces.